src/pages/help-centre/_utility.ts

```diff
@@ -24,7 +26,7 @@ export const splitArticles = (array: ArcticlesType[], length: number): string[]
 
 export const euArticles = (array: string[]): [string, string[], string[]] => {
     const second_array = [
-        ...array[1].filter((el) => el.category.props.translate_text !== 'Deriv X'),
+        ...array[1].filter((el) => el.category.props.translate_text !== eu_discards[0]),
```

Comment:
```
eu_discards[0]
```

Will only check on the first element of the array. Do you think we can loop over to this `eu_discards` ?

Reply:
In this implementation we see array[0], array[1], array[2] by groups of topics. Here we just delete 1 specific topic from specific array. In each array only 3 different topics

It seems to me that make sense use loop in big arrays. Let me know if we need to do it after my explanation 🙏